node/src/web.rs

```diff
@@ -158,12 +173,19 @@ async fn get_public_data(state: State<WebServerState>) -> Json<StaticWebData> {
 pub async fn start_web_server(
     root_task_handle: Arc<crate::tracking::TaskHandle>,
     signature_debug_request_sender: broadcast::Sender<SignatureDebugRequest>,
-    config: WebUIConfig,
+    config: WebUIConfig, // assuming this contains host and port
     static_web_data: StaticWebData,
     contract_state_receiver: watch::Receiver<ProtocolContractState>,
 ) -> anyhow::Result<BoxFuture<'static, anyhow::Result<()>>> {
     use futures::FutureExt;
 
+    // Print out the config parameters before attempting to bind
+    tracing::debug!(
+        "Attempting to bind web server to host: {}, port: {}",
+        config.host,
+        config.port
+    );
```

Comment:
Consider using structured logging here instead of string interpolation. Structured fields make logs more searchable and filterable in log aggregation systems. You'll be able to query for specific hosts or ports without parsing the message string.

https://docs.rs/tracing/latest/tracing/#recording-fields

```suggestion
    tracing::info!(
        host = %config.host,
        port = %config.port,
        "Attempting to bind web server to host",
    );
```